Automattic/vip-go-mu-plugins
# Feature: Add enabling or disabling by ID

## Description
It would be nice to be able to selectively enable or disable a feature by an array of ids. This PR also gets rid of the `site_id()` method since we can use the Constant_Mocker class now.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Sandbox into a site
2) Edit in `$feature_ids` with `[ 'feature' => [ <your-site-id> => true ] ]`
3) shell in and do `\Automattic\VIP\Feature::is_enabled_by_ids( 'feature' )` and expect it to return true
4) Repeat step 2 but replace `<your-site-id>` with random numbers
5) shell in and do `\Automattic\VIP\Feature::is_enabled_by_ids( 'feature' )` and expect it to return false
